frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -780,6 +795,25 @@ export function ContainersSection(props: { resource: KubeObjectInterface | null
           ))
         )}
       </>
+
+      {!initContainers ? (
+        <SectionBox>
+          <Empty>{t('resource|No data to be shown.')}</Empty>
+        </SectionBox>
+      ) : (
+        <>
+          <SectionBox title={t(`resource| Init Containers`)} />
+          {initContainers.map((initContainer: any, i: number) => (
+            <SectionBox key={i} outterBoxProps={{ pt: 1 }}>
```

Comment:
I know you just used what's written for the regular containers, but the `key` should be textual, so `key={`init_container_${i}`}` is better; and `initContainer` should have the proper type (`KubeContainer` I think).

Reply:
Feel free to also modify the regular container's key+type in a different commit if you want.